front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -292,6 +292,25 @@ export function DocumentView({
   );
 }
 
+const timestampFromTimeSettings = function (
```

Comment:
NIT: As the file is getting quite big, maybe move that function in an util file for the Chat? (feel free to discard of course)

Reply:
Also NIT, do you agree it would be easier to read if it was called `getMinTimestampFromTimeSettings`?

Reply:
Agree that the file is too big => but would require semantic cutting, not just moving this fn (I think the best way would be moving 3-4 big components out) => for another PR
Agree for the renaming